editor/modes/text-editor/style.scss

```diff
@@ -59,40 +87,73 @@
 	line-height: 150%;
 	transition: padding .2s linear;
 
+	@include break-small() {
+		padding-top: 60px + $admin-bar-height-big;
+	}
+
+	@include break-medium() {
+		padding-top: 60px + $admin-bar-height;
+	}
+
 	&:focus {
 		box-shadow: none;
 	}
 }
 
 /* Use padding to center text in the textarea, this allows you to click anywhere to focus it */
-.editor-text-editor__formatting,
 .editor-text-editor {
 	padding-left: $item-spacing * 2;
 	padding-right: $item-spacing * 2;
-	max-width: 1440px;
+	max-width: $break-huge;
 	margin: 0 auto;
-}
 
-@media only screen and ( min-width: 960px ) {
-	.editor-text-editor__formatting,
-	.editor-text-editor {
+	@include break-large() {
 		padding-left: 10%;
 		padding-right: 10%;
 	}
-}
 
-@media only screen and ( min-width: 1280px ) {
-	.editor-text-editor__formatting,
-	.editor-text-editor {
+	@include break-wide() {
 		padding-left: 15%;
 		padding-right: 15%;
 	}
-}
 
-@media only screen and ( min-width: 1440px ) {
-	.editor-text-editor__formatting,
-	.editor-text-editor {
+	@include break-huge() {
 		padding-left: 200px;
 		padding-right: 200px;
 	}
 }
+
+.editor-text-editor__formatting {
+	&:before,
+	&:after {
+		content: "";
+		display: block;
+		margin: 0;
+		flex: 0 1 $item-spacing * 2;
+	}
+
+	&:after {
+		margin-left: auto;
+	}
+
+	@include break-large() {
+		&:before,
+		&:after {
+			flex: 0 1 10%;
+		}
+	}
+
+	@include break-wide() {
+		&:before,
+		&:after {
+			flex: 0 1 15%;
```

Comment:
Just realized that a) you're right, width could have worked just as well here, and b) this doesn't actually solve the problem I was trying to solve. I'm going to keep working for a bit here. I must have lacked more sleep than I thought I lacked. Thanks for the reviews and apologies, going to bake this one a bit more.